src/avram/tasks/gen/migration.ecr

```diff
@@ -3,7 +3,10 @@ class <%= migration_class_name %> < Avram::Migrator::Migration::V1
     <%- if formatted_migrate_contents -%>
     <%= formatted_migrate_contents %>
     <%- else -%>
-    # create :things do
+    # Read more on migrations
+    # https://www.luckyframework.org/guides/database/migrations
```

Comment:
I'm not sure if this will get too busy after time, but I think it's worth adding and seeing how it does! Maybe we have a "no comments" config option in the future that people can use once they're comfortable

Reply:
Yeah. I don't like when it gets too busy, but I wanted to make sure it was clear that the second you open the file and you're like "ok, now what?" you at least have a direction to go... I have an idea though....

Reply:
I kind of like `.luckyconfig` more than .rc but yeah I think that is an awesome idea